internal/controller/resourcefieldexport/controller.go

```diff
@@ -110,7 +110,9 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 	if err != nil {
 		return r.degradedStatus(ctx, fieldExports, fmt.Errorf("failed to write to destination: %s", err))
 	}
+	logger.Info(fmt.Sprintf("Successfully written to %s %s", string(fieldExports.Spec.To.Type), fieldExports.Spec.To.Name))
```

Comment:
The structured logger takes variadic parameters like explained here

https://pkg.go.dev/github.com/go-logr/logr#hdr-Usage

```suggestion
	logger.Info("output written to", "type",  fieldExports.Spec.To.Type, "name", fieldExports.Spec.To.Name)
```